crime-analytics.ipynb

```diff
@@ -60,10 +60,8 @@
   },
```

Comment:
<p>This seems like we could remove this.</p>

 _Reply via <a href='https://app.reviewnb.com/cmenguy/crime-analytics/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>